### DISCUSSION TITLE
Cant make it work! HELP!

### DISCUSSION THREAD

**Discussion by UncleSunn** [emailjs-com/help#66] [Q&A]
Hello there everyone, 

I'm facing a problem that I can't seem to solve by myself and I have to say that I really tried everything I know.
Step by step video tutorials and detailed instructions from the instructions at [**_EmailJS Docs._**](https://www.emailjs.com/docs)

I will provide you below screenshots of my code and hopefully someone maybe can spot anything that might resolve my problem!
Maybe there is something that I'm missing I don't know..!

_**Thank you very much if you take the time to take a look at the code! Trust me I need every help I can find so thanks in advance!!!**_

• The first screenshot is from the form.js the javascript code.
![3](https://github.com/emailjs-com/help/assets/149025295/4e9a2148-a12a-4e20-ae85-f6228eeedd6c)

• The second screenshot is from the <head> section of my html code.
 ![2](https://github.com/emailjs-com/help/assets/149025295/a3b10119-e159-46c5-a4b9-7e4085bb6ff7)

• The last screenshot is from the form HTML code.
![1](https://github.com/emailjs-com/help/assets/149025295/051bd912-799c-4b09-8eac-2f13c3941c41)

**Comment by xr0master** [comment]
Your screenshots look good. Probably, in your case, you need to use `params` instead of `templateParams`.
What's not working for you?

**Reply by UncleSunn** [reply]
Yeah I used accidently an old screenshots where I had both as you can see but I've changed those to params but still no luck. I re did all the steps again and again 5 times but nothing yet. Anything else you think might help here?

**Reply by xr0master** [reply]
I suggest you describe what doesn't work and what response you get.
